app/controllers/icons_controller.rb

```diff
@@ -49,6 +49,20 @@ def show
       @posts = posts_from_relation(Post.where(where_calc).ordered)
     elsif params[:view] == 'galleries'
       use_javascript('galleries/expander_old')
+    else
```

Comment:
Can you put an explanatory comment for this chain of the if statement? Something like:

```rb
# Approximate icon counts for non-owner
# Check only more obviously visible posts for icon counts
# (public posts, and constellation-only posts if logged in)
```